tobiasHe/assignments/treesBFS/lc102/lc102.js

```diff
@@ -0,0 +1,91 @@
+// Problem Statement #
+
+// Given a binary tree, populate an array to represent its level-by-level traversal. You should populate the values of all nodes of each level from left to right in separate sub-arrays.
+
+class TreeNode {
+	constructor(value) {
+		this.value = value;
+		this.left = null;
+		this.right = null;
+	}
+}
+/* 
+  using BFS to traverse the tree, while keeping the values of each level
+  initialize a queue with root
+  keep track of queueLength, to move on to next levels
+  shift out the queue and check if current.value has any left and right children
+  if there are any, add them to the queue 
```

Comment:
Nice overview but missing a step about adding the current level's values to the current Level array.